java/src/jmri/jmrix/loconet/LnSensor.java

```diff
@@ -126,7 +140,7 @@ public void message(LocoNetMessage l) {
 
     @Override
     public void dispose() {
-        tc.removeLocoNetListener(~0, this);
+        //tc.removeLocoNetListener(~0, this);
         super.dispose();
     }
 
```

Comment:
Can remove `dispose()` method from LnSensor and allow parent object's method do its work "behind the scenes".